src/backend/distributed/commands/create_distributed_table.c

```diff
@@ -1261,13 +1261,13 @@ CopyLocalDataIntoShards(Oid distributedRelationId)
 
 		if (rowsCopied % 1000000 == 0)
 		{
-			ereport(DEBUG1, (errmsg("Copied %ld rows", rowsCopied)));
```

Comment:
When I look at the diff this looks like you made a mistake, even though what you're really doing is fixing a mistake.

For future readers, maybe make a note in the commit message that you also fixed some inconsistent usage of `%ld` and `%lu`?

Reply:
added to commit log